packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.component.tsx

```diff
@@ -104,17 +103,15 @@ export const DobField: React.FC = () => {
             <OpenmrsDatePicker
               id="birthdate"
               {...birthdate}
-              dateFormat={dateFormat}
               onChange={onDateChange}
               maxDate={today}
               labelText={t('dateOfBirthLabelText', 'Date of Birth')}
-              invalid={!!(birthdateMeta.touched && birthdateMeta.error)}
-              invalidText={birthdateMeta.error && t(birthdateMeta.error)}
+              isInvalid={!!(birthdateMeta.touched && birthdateMeta.error)}
               value={birthdate.value}
-              carbonOptions={{
-                placeholder: placeHolder,
-              }}
             />
+            {!!(birthdateMeta.touched && birthdateMeta.error) && (
+              <div className={styles.radioFieldError}>{birthdateMeta.error && t(birthdateMeta.error)}</div>
+            )}
```

Comment:
Shouldn't we also make the error field part of the component itself?

Reply:
Yes... Upcoming feature.